<?xml version="1.0"?>
<launch>
    <param name="/use_sim_time" value="true" />
    <arg name="vocabularty_path" default="$(find fisheye_orb_ros)/config/Vocabulary/ORBvoc.bin" />
    <!--arg name="camera_setting_path" default="$(find fisheye_orb_ros)/config/xinli_stereo137.yaml" /-->
    <arg name="camera_setting_path" default="$(find fisheye_orb_ros)/config/stereo_robot.yaml" />
    <arg name="bUseViewer" default="true" />
    <arg name="bEnablePublishROSTopic" default="true" />
    <arg name="SAVEorLOAD" default="SAVE" />
    <arg name="mapname" default="/media/shinan/DATA/ROSbag/stereo_robot/newroom.bin" />
    <arg name="left_topic" value="/left/image_raw/compressed"/>
    <arg name="right_topic" value="/right/image_raw/compressed"/>
    <!--arg name="left_topic" value="/object_detection_2D_node_combine/left_image_raw"/>
    <arg name="right_topic" value="/object_detection_2D_node_combine/right_image_raw"/-->
    <arg name="BagPath" default="/media/shinan/DATA/ROSbag/data_office2/" />

    <node name="rostopic_stereo_tf" pkg="fisheye_orb_ros" type="rostopic_stereo_tf" output="screen" launch-prefix="gdb -ex run --args"
    	args="$(arg vocabularty_path) $(arg camera_setting_path) $(arg SAVEorLOAD) $(arg BagPath) $(arg mapname) $(arg left_topic) $(arg right_topic) ">
  	</node>
</launch>
